packages/truffle-debugger/lib/session/index.js

```diff
@@ -1,12 +1,16 @@
 import debugModule from "debug";
+//returns true on success, false on already unloadee
```

Comment:
typo

Reply:
(what is this comment even doing here?)

Reply:
Whoa. Looks like an accidental paste. Deleted.